src/FondOfCodeception/Module/Spryker.php

```diff
@@ -64,11 +64,11 @@ public function _initialize()
 
         $this->initEnvironment();
 
-        if (trim($this->config[SprykerConstants::CONFIG_GENERATE_TRANSFER]) === true) {
+        if (trim($this->config[SprykerConstants::CONFIG_GENERATE_TRANSFER]) == true) {
```

Comment:
cast would be better instead of this compare

Reply:
@julianzimmermann done